pkgs/development/python-modules/duckdb-engine/default.nix

```diff
@@ -76,10 +78,10 @@ buildPythonPackage rec {
     # user agent not available in nixpkgs
     "test_user_agent"
     "test_user_agent_with_custom_user_agent"
+    # Number of examples > settings.maxExamples
+    "test_simple_string"
```

Comment:
Is there an issue for this on the upstream repo?

Reply:
After looking at the upstream, the test was fine but generated output that looked like an error. Dropped these two lines and it works.

Reply:
Ok great.